skyportal/handlers/api/galaxy.py

```diff
@@ -104,11 +105,37 @@ def get(self, catalog_name=None):
           tags:
             - galaxies
           parameters:
-            - in: catalog_query
-              name: name
+            - in: query
+              name: catalog_name
               schema:
                 type: string
               description: Filter by catalog name (exact match)
+            - in: query
+              name: localizationDateobs
+              schema:
+                type: string
+              description: |
+                Event time in isot format
```

Comment:
```suggestion
                Event time in ISO 8601 format (`YYYY-MM-DDTHH:MM:SS.sss`).
```

What timezone is this, or is that not relevant?

Reply:
UTC is implied.